cli/ballerina-cli/src/main/java/io/ballerina/cli/cmd/PackCommand.java

```diff
@@ -177,6 +180,42 @@ public void execute() {
             if (buildOptions.dumpBuildTime()) {
                 BuildTime.getInstance().projectLoadDuration = System.currentTimeMillis() - start;
             }
+
+            Path readmeFilePath = null;
+
+            // 1. Check if a custom readme path is defined in Ballerina.toml
+            Optional<String> readmePathOptional = project.currentPackage().manifest().readme();
```

Comment:
The `project.currentPackage().manifest().readme()` method returns a string. Hence, the `readmePathOptional` should be of type `String`